client/landing/stepper/declarative-flow/internals/components/deferred-render/use-deferred-render/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+interface UseDeferredRenderProps {
+	timeMs: number;
+}
+
+export const useDeferredRender = ( { timeMs }: UseDeferredRenderProps ) => {
+	const [ isReadyToRender, setIsReadyToRender ] = useState( false );
+	const timeoutId = useRef< number | null >( null );
+
+	useEffect( () => {
+		const id = setTimeout( () => {
```

Comment:
This will change the ID type to number
```suggestion
		const id = window.setTimeout( () => {
```

Reply:
Nice tip!

Fixed here [eb5cbf1](https://github.com/Automattic/wp-calypso/pull/97519/commits/eb5cbf1994d151d50a631ebcfd9c36cc1ef7d4b5)

Reply:
I mentioned it to remove the need for `timeoutId.current = id as unknown as number;` :D

Reply:
Ops! Fixed here 🤦  [24f8ad4](https://github.com/Automattic/wp-calypso/pull/97519/commits/24f8ad4b62d9af57f0b8f8946c3f421a9acd2683)